Dockerfile.template

```diff
@@ -17,4 +17,4 @@ VOLUME /mnt/boot \
 CMD [ "/entry.sh" ]
 
 # avoid display crash on some host systems
-RUN rm /lib/systemd/system/sysinit.target.wants/systemd-udev-trigger.service
+RUN rm /lib/systemd/system/sysinit.target.wants/systemd-udev-trigger.service
```

Comment:
Can you tidy this commit by restoring the empty lines at the end of these files?